pkg/scheduler/eventhandlers_test.go

```diff
@@ -334,3 +342,82 @@ func TestPreCheckForNode(t *testing.T) {
 		})
 	}
 }
+
+// Test for informers of resources we care about is registered
+func TestAddAllEventHandlers(t *testing.T) {
+	tests := []struct {
+		name              string
+		gvkMap            map[framework.GVK]framework.ActionType
+		expectInformerNum int
```

Comment:
Maybe separate it with staticInformerNum and dynamicInformerNum?

Reply:
ok